README.md

```diff
@@ -61,7 +61,18 @@ File: C:\Templates\azuredeploy.json
 #### SARIF
 Results are written to the file specified (with the `-o` or `--output-file-path` argument) in [SARIF](https://sarifweb.azurewebsites.net) format.
 
-### Understanding and Customizing Rules
+#### Error codes
+| Scenario      | Exit Code |
+| ----------- | ----------- |
+| Success: Operation is successful | 0 |
+| Error: GenericError | 1 |
```

Comment:
Maybe it's worth reordering the list such that this one is at the end, and expressing that it corresponds to any error not previously mentioned?